bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/igrins2/Igrins2Recipe.java

```diff
@@ -261,11 +269,45 @@ private SpectroscopyResult calculateSpectroscopy(
         calcSource.sed.accept(specS2N);
 
         final SpecS2N[] specS2Narr = new SpecS2N[]{specS2N};
+
+        VisitableSampledSpectrum finalS2NSpectrum = specS2N.getFinalS2NSpectrum();
+        double snr = finalS2NSpectrum.getY(SnrWavelength);
+        Log.fine(String.format("S/N @ %.1f nm = %.3f", SnrWavelength, snr));
+
         return new SpectroscopyResult(p, instrument, IQcalc, specS2Narr, slit, throughput.throughput(), altair,
                 Option.apply(new ExposureCalculation(exposureTime, numberExposures, snr)));
```

Comment:
As discussed offline we may need to enhance this to support a set of calculations in case we have multiple detectors as for IGRINS-2